### ISSUE TITLE
Problem with getting started

### ISSUE THREAD

**Issue by vpnels** [gatsbymanor/gatsby-theme-massively#1]
I followed the Gatsby Manor Getting Started. I unmounted the theme "Starter", and got and mounted the theme "Massively". When I do a "gatsby develop" I see the following errors:
/////
src/css/noscript.css
undefined [undefined]

src/css/main.css
undefined [undefined]
/////

The requested files are in src/css.

What do I need to do?

I can see the site with the theme at localhost:8000. However, I don't know if I'm missing something.

Aloha,

Pascal

**Comment by nodox** [comment]
@vpnels thanks for raising this issue! I did the starter from scratch and was not able to reproduce. 😓 

Maybe you have an older version? Try a fresh install. 

Can you send me the steps to reproduce or send me the docker image of your env with the error?

Best,
Steven

**Comment by vpnels** [comment]
Hi Steven,

Thank you so much for answering so quickly.

I did the install of everything gatsby within the last 2 days. I installed the gatsby-theme-kit just today. I have done the Getting Started - Starting from Scratch steps 3 separate times today, on 3 new, clean directories.

Here is what I’m seeing when I run gatsby develop:



Aloha from the Big Island of Hawaii,

Pascal

**Comment by nodox** [comment]
@vpnels it seems you forgot to attach an error log or picture? 😅  Maybe try again.

Greetings from the Big Apple that is New York!
😄

**Comment by vpnels** [comment]
I did attach a picture. It didn’t get to you?

Here is what I’m seeing:

Orion:gatsby-theme-kit Pascal$ gatsby develop
success delete html files from previous builds — 0.010 s
success open and validate gatsby-config.js — 0.004 s
success copy gatsby files — 0.142 s
success onPreBootstrap — 0.006 s
success source and transform nodes — 0.183 s
success building schema — 1.705 s
success createLayouts — 0.008 s
success createPages — 0.103 s
success createPagesStatefully — 0.012 s
success onPreExtractQueries — 0.001 s
success extract queries from components — 0.173 s
success run graphql queries — 0.007 s
success write out page data — 0.003 s
success write out redirect data — 0.001 s
success update schema — 0.080 s
success onPostBootstrap — 0.001 s

info bootstrap finished - 16.215 s


src/css/noscript.css
undefined [undefined]


src/css/main.css
undefined [undefined]

 DONE  Compiled successfully in 8608ms                                7:39:43 PM

 I  Your site is running at http://localhost:8000
 I  Your graphql debugger is running at http://localhost:8000/___graphql
 WAIT  Compiling...                                                   7:39:43 PM

 DONE  Compiled successfully in 69ms                                  7:39:44 PM

 I  Your site is running at http://localhost:8000
 I  Your graphql debugger is running at http://localhost:8000/___graphql
>

**Comment by vpnels** [comment]
<img width="682" alt="screen shot 2018-02-22 at 19 41 29" src="https://user-images.githubusercontent.com/8770790/36580217-6cabe3b0-180b-11e8-9f1e-f2ed461578f3.png">
Ah! I think I know why you didn't get the attached picture. I was replying via email, rather than from the github interface.

This is what I'm seeing as errors:
 
src/css/noscript.css
undefined [undefined]

src/css/main.css
undefined [undefined]

**Comment by nodox** [comment]
Can you upload your project on Github so I can clone and investigate?

**Comment by vpnels** [comment]
Steven,

I am on MacOS. I know nothing about Docker. I know what a VM is, but I have no idea how to put this into a VM on MacOS.

Aloha,

Pascal

**Comment by nodox** [comment]
Can you upload your code and point me to a Github repo with your code?

**Comment by vpnels** [comment]
I think that I can manage with github. I'll try. May take me a while.

Pascal

**Comment by vpnels** [comment]
Steven,

The project is at https://github.com/vpnels/test_massively

I did NOT upload the node_modules directory.

Pascal

**Comment by nodox** [comment]
tl;dr Don't worry about it. You've got the whole theme.

After some revision and reproduction, if you are able to get the styles on the page then you have the complete theme. You aren't missing anything and unless it breaks the build or throws a localhost:8000 error where the screen refers to the offending line in CSS then you should be fine. Let me know if it breaks 😅

**Comment by vpnels** [comment]
Thanks Steven. I'll continue.
